src/component/LayerCarousel/LayerCarousel.tsx

```diff
@@ -159,11 +163,17 @@ export default class LayerCarousel extends React.Component<LayerCarouselProps, L
     }
     const currentlyVisibleLayer = this.props.layers.find(l => l.getVisible());
     this.setState({
+      // TODO Don't access private property ol_uid
+      // @ts-ignore
       originalBaseLayerOlUid: currentlyVisibleLayer ? currentlyVisibleLayer.ol_uid : undefined
```

Comment:
There is a ol util method to acces the ol_uid:
https://openlayers.org/en/latest/apidoc/module-ol_util.html#.getUid

Reply:
Will be fixed in a follow up PR.